lua/LUA/ak/demo-anlagen/ampel/Ampeldemo-Grundmodelle-main.lua

```diff
@@ -205,8 +206,11 @@ end
 -- endregion
 
 local ModuleRegistry = require("ak.core.ModuleRegistry")
-ModuleRegistry.registerModules(require("ak.core.CoreLuaModule"), require("ak.data.DataLuaModule"),
-                               require("ak.road.CrossingLuaModul"))
+ModuleRegistry.registerModules(
+    require("ak.core.CoreLuaModule"),
+    require("ak.data.DataLuaModule"),
+    require("ak.road.CrossingLuaModul")
+)
```

Comment:
Diese einzelne Änderung widerspricht leider den in `/lua-format.conf` hinterlegten Formatierungsrichtlinien, darum würde ich diese nicht übernehmen wollen.

Kannst Du den Code vor dem Pull-Request mit lua-format formatieren?

Reply:
Das geht zum Beispiel in VSCode mit dem Plugin koihik.vscode-lua-format
Oder mit Luarocks https://github.com/Koihik/LuaFormatter

Reply:
Diese Formatierungsänderung ist natürlich nicht wichtig (auch wenn mir eine mehrzeilige Schreibweise besser lesbar vorkommt). Kannst du sie herausnehmen oder soll ich einen neuen Commit (für das Original-Coding) machen?

VSCode und/oder Luarocks müsste ich erst wieder installieren - bislang verwende ich einfach notepad++

Reply:
Ein eigener Commit in diesem Pull-Request wäre toll, dann kann ich das Ganze einfach mit einem Klick übernehmen.

Reply:
Mist, jetzt waren mir die nicht zu publizierenden Änderungen mit in den commit&push gerutscht:
lua/LUA/ak/third-party/json.lua 
lua/LUA/ak/third-party/json_original.lua
web-app/package-lock.json
web-server/package-lock.json 
... und ich musste sehen wie ich diesen commit wieder zurücknehmen konnte.

Der revert hat wohl geklappt.

Reply:
Sehr schön!